src/SignupModal.jsx

```diff
@@ -26,6 +26,34 @@ export const SIGNUP_MODAL_EMAIL_CLASS = `${SIGNUP_MODAL_CLASS}-email`;
  * @returns {React.element} SignupModal
  */
 export const SignupModal = ({ onDismiss, signupOptions, ...other }) => {
+	const facebookOnClick = () => {
```

Comment:
The functions could be consolidated into one as well, ex:
```jsx
const gaTracking = type => {
  window.dataLayer = window.dataLayer || [];
  window.dataLayer.push({
    event: 'registrationStart',
    form: {
      type
    },
  });
}
```

Reply:
It would need to return a handler though, something like
```js
const gaTracking = type => () => {
  window.dataLayer = window.dataLayer || [];
  window.dataLayer.push({
    event: 'registrationStart',
    form: {
      type
    },
  });
}
```
and used like:
```html
<a onClick={gaTracking('facebookAuthButton'}>...</a>
<a onClick={gaTracking('email'}>...</a>
<a onClick={gaTracking('googleAuthButton'}>...</a>
```

Reply:
ahhh good idea! i'll do that instead